### PR TITLE
[Doc Rework] Create admin.md

### PR DESCRIPTION
**Pull request by mapedraza**
This PR merge `installationguide.md` and `operations.md` into `admin.md` file.

This PR is part of a bigger series #1329

Note this PR points towards `task/doc-rework` branch

### CODE REVIEW

File: doc/admin.md [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+-   [Alarms](#alarms)
+
+The **IoT Agent node library** is not a standalone product and should be added as a dependency to `package.json` of the
+project is going to be used in. The library is published in the NPM repository, so it can be added to the project by
+adding the following line to the `package.json` file:
+
+```json
+...
+"dependencies": {
+    "iotagent-node-lib": "*",
```
Comment (fgalan) [inline doc/admin.md:47]: This is a kind o "blind" dependency (meaning "the last version of the library, whatever it is") and I think it is not the recomended way...

What about?

```suggestion
    "iotagent-node-lib": "x.y.z",
```

explaning just below that `x.y.z` means an actual library version.

File: doc/api.md [hunk 2]
```diff
@@ -2,66 +2,6 @@
 
 <!-- TOC -->
 
--   [Preface](#preface)
```
Comment (fgalan) [inline doc/api.md:5]: Is this removal of the TOC intentional?

File: doc/operations.md [hunk 3]
```diff
@@ -1,127 +0,0 @@
...
-
-Every error has a code composed of a prefix and an ID, codified with the following table:
-
-| Prefix           | Type of operation                                          |
-| :--------------- | :--------------------------------------------------------- |
-| MONGODB          | Errors related with the MongoDB repository                 |
-| IOTAM            | Errors related with the IoTA Manager                       |
-| KEYSTONE         | Errors related with trust token retrieval                  |
-| ORION            | Errors in Context Broker access                            |
-| VALIDATION-FATAL | Errors related with management of the Validation templates |
```
Comment (fgalan) [inline doc/operations.md:127]: Does this section exists in the new document?

File: doc/api.md (hunk 2 above)
Comment (mapedraza) [reply doc/api.md]: Restored here: https://github.com/telefonicaid/iotagent-node-lib/pull/1369/commits/a1cb5da739231fb75b1bcdec62c6797e133414a1

File: doc/admin.md [hunk 4]
```diff
@@ -0,0 +1,565 @@
...
+project is going to be used in. The library is published in the NPM repository, so it can be added to the project by
+adding the following line to the `package.json` file:
+
+```json
+...
+"dependencies": {
+    "iotagent-node-lib": "x.y.z",
+}
+```
+
```
Comment (fgalan) [inline doc/admin.md:50]:
```suggestion

where `x.y.z` is a particular version of the library (of the ones included [here](https://www.npmjs.com/package/iotagent-node-lib?activeTab=versions))

```

In addition, the following fragment could be include (I cannot use it in the suggestion due to the usage of ```):

---------

As alternative, you can use the master branch as dependency. In this case, you will be using the latest version of the code but note that some instability could exist (as the code in master is work in progress until the next version is closed).

```json
...
"dependencies": {
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
}
```

Comment (mapedraza) [reply doc/admin.md:50]: Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1369/commits/6a9bfa02a72deba33617399dd87bd145429a8c18

File: doc/operations.md (hunk 3 above)
Comment (fgalan) [reply doc/operations.md:127]: NTC

File: doc/api.md [hunk 5]
```diff
@@ -440,9 +440,9 @@ depending on the JEXL expression evaluation:
 
 This is a flag that can be enabled by activating the parameter `appendMode` in the configuration file or by using the
 `IOTA_APPEND_MODE` environment variable (more info
-[here](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md)). If this flag is
-activated, the update requests to the Context Broker will be performed always with APPEND type, instead of the default
-UPDATE. This have implications in the use of attributes with Context Providers, so this flag should be used with care.
+[here](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/admin.md)). If this flag is activated, the
```
Comment (fgalan) [inline doc/api.md:443]:
```suggestion
[here](admin.md)). If this flag is activated, the
```

File: doc/admin.md [hunk 6]
```diff
@@ -0,0 +1,574 @@
...
```
Comment (fgalan) [inline doc/admin.md:50]:
```suggestion

wher `x.y.z` is an actual version number.

```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**